.github/workflows/run-secrets-detection.yml

```
@@ -0,0 +1,40 @@
+name: Secrets Detection
+on: pull_request
+
+jobs:
+  secrets_detection:
+    runs-on: ubuntu-latest
+    if: github.repository == 'demisto/content'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: add_triger_secrets_run
```

This will be changed before merge.
```suggestion
          ref: master
```